File: C_Projects/C_Euler/src/1.c
#include "C_Euler.h"

int64 Euler_1()
{
    int count;
    int sum = 0;
    for (count = 1; count < 1000; count++) {
        if ((count % 3 == 0) || (count % 5 == 0))
            sum += count;
    }
    return (int64) sum;
}

File: Py_Projects/Euler/1.py
def Euler_1(top=1000):
	'''Find the sum of all the multiples of 3 or 5 below the input
	'''
	total = 0
	for x in range(top):
		if x % 5 == 0 or x % 3 == 0:
			total += x
	return total

if __name__ == '__main__':
	print(Euler_1())
